modules/analysis-common/src/main/java/org/elasticsearch/analysis/common/LegacyRomanianStemmer.java

```diff
@@ -0,0 +1,744 @@
+/*
+ * @notice
+ * Generated by Snowball 2.0.0 - https://snowballstem.org/
+ *
+ * Modifications copyright (C) 2024 Elasticsearch B.V.
+ */
+
+package org.elasticsearch.analysis.common;
+
+import org.tartarus.snowball.Among;
+
+/**
+* This class implements the stemming algorithm defined by a snowball script.
+* NOTE: This is the RomanianStemmer used in Lucene 9
+*
+* @deprecated this class exists for backwards compatibility of indices created with Lucene 9
```

Comment:
I don't think we should do this. We aren't providing bwc layer for ANY other stemmer, its weird that we are for this one.